rrmngmnt/network.py

```diff
@@ -545,3 +545,18 @@ def delete_interface(self, interface):
             logger.error(e)
             return False
         return True
+
+    def get_mac_by_ip(self, ip):
+        """
+        Get mac address by ip address
+
+        :param ip: ip address
+        :type ip: str
+        :return: mac address
+        :rtype: str
+        """
+        try:
```

Comment:
why try/except? these methods dont throw
I prefer that you actually check what is returned and act accordingly